common/taskana-common/src/main/java/pro/taskana/common/api/WorkingTimeSchedule.java

```diff
@@ -0,0 +1,74 @@
+package pro.taskana.common.api;
+
+import java.time.DayOfWeek;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.EnumMap;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Set;
+import java.util.SortedSet;
+import java.util.TreeSet;
+
+class WorkingTimeSchedule {
+
+  private final Map<DayOfWeek, SortedSet<LocalTimeInterval>> workingTimeByDayOfWeek =
+      new EnumMap<>(DayOfWeek.class);
```

Comment:
Persist both ways of sorting (ASC + DESC), so that we only have to sort during instantiation